malariagen_data/anopheles.py

```diff
@@ -7236,8 +7290,8 @@ def ihs_gwss(
         gap_scale: ihs_params.gap_scale = ihs_params.gap_scale_default,
         include_edges: ihs_params.include_edges = True,
         use_threads: ihs_params.use_threads = True,
-        min_cohort_size: base_params.min_cohort_size = 15,
-        max_cohort_size: base_params.max_cohort_size = 50,
+        min_cohort_size: Optional[base_params.min_cohort_size] = 15,
+        max_cohort_size: Optional[base_params.max_cohort_size] = 50,
```

Comment:
Suggest defining these defaults on the `ihs_params` class, i.e., `ihs_params.min_cohort_size_default` and `ihs_params.max_cohort_size_default`.